test/end-to-end/src/end2end-tests.ts

```diff
@@ -0,0 +1,86 @@
+import { ClipboardCheckmark16Filled } from "@fluentui/react-icons";
```

Comment:
Well, we still want a descriptive name for the file, especially since the whole path is not visible in VSC when the file is being edited. What would be a better name for it?

Reply:
By that logic every file name should include its whole path in order to completely identify it.  Hovering over the opened file name gives you the path and activating that file window automatically expands and highlights that file in the file tree giving you the context.  I think the existing name of "ui-test" is sufficient.

Reply:
Unfortunately, there's another file named "ui-test", so we can't use that. How about "host-tests"? 

BTW, what is the harm in having a file name that repeats terms from the path?

Reply:
> By that logic every file name should include its whole path in order to completely identify it. Hovering over the opened file name gives you the path and activating that file window automatically expands and highlights that file in the file tree giving you the context. I think the existing name of "ui-test" is sufficient.

No, I'm not saying every file name should include its own path. I'm saying it is OK for a filename to repeat some information that is in the path in the interest of having a descriptive file name. It is a good thing if users (and by "users' here, we're talking about **my team**), can see at a glance the basic purpose of the file without having to hover over it. What is the harm in having a file name that repeats terms from the path? 

At any rate, there's another file named "ui-test", so we can't use that. How about "host-tests"?

Reply:
"host-tests" sounds like a descriptive name to me.  

You might also consider just including the code in the test.index.tsx file since that is the only place that it is referenced.

Reply:
I renamed it to "host-tests".